server/models/Update/equity_accounts.sql

```diff
@@ -0,0 +1,2 @@
+ UPDATE account SET type_id = 3
+ WHERE number LIKE '1%' AND type_id <> 6;
```

Comment:
Could you do the same operation with data for assets and liabilities, to be sure that all accounts have the correct type.

It is important for correctly test what we have in our Bilan

Reply:
I need the assets and liabilities conditions for updating those accounts

Reply:
You can use these document for seeing which account is for assets (actifs) and liabilities (passifs)

## Assets : 
![actifs](https://user-images.githubusercontent.com/5445251/42878821-55356cfe-8a86-11e8-8dd0-d435847747c3.jpg)

## Liabilities
![passifs](https://user-images.githubusercontent.com/5445251/42878848-6ddf6c6e-8a86-11e8-9da3-4637270d0b3c.jpg)